core/new-gui/src/app/workspace/component/property-editor/property-editor.component.ts

```diff
@@ -62,13 +62,30 @@ export class PropertyEditorComponent implements OnInit {
    */
   registerHighlightEventsHandler() {
     merge(
-      this.workflowActionService.getJointGraphWrapper().getJointOperatorHighlightStream(),
-      this.workflowActionService.getJointGraphWrapper().getJointOperatorUnhighlightStream(),
-      this.workflowActionService.getJointGraphWrapper().getJointGroupHighlightStream(),
-      this.workflowActionService.getJointGraphWrapper().getJointGroupUnhighlightStream(),
-      this.workflowActionService.getJointGraphWrapper().getLinkHighlightStream(),
-      this.workflowActionService.getJointGraphWrapper().getLinkUnhighlightStream(),
-      this.workflowVersionService.workflowVersionsDisplayObservable()
```

Comment:
seems this is removed by mistake?!

Reply:
@Jiyang-Wu please reformat the file and make sure you resolve the merge conflicts correctly. I've fixed some wrong fix in `app.module.ts`.  This block's format is off thus hard to eyeball the conflicts.

Reply:
Conflict has been fixed.

Reply:
Sorry, I still don't see this line`this.workflowVersionService.workflowVersionsDisplayObservable()`